### ISSUE TITLE
create a Where's Waldo finder and generator?

### ISSUE THREAD

**Issue by robertleeplummerjr** [BrainJS/brain.js#53]
This came to me talking about the neural net when explaining it to a friend, might be fun.

**Comment by wagenaartje** [comment]
Fun, but very hard! I think the way to go on this is to split the big image into smaller chunks (say 100x100 each) and then run a convolutional net over each one of those chunks. Let it classify if waldo is present. A generator is a whole other story though...

**Comment by robertleeplummerjr** [comment]
Agreed!  Progress for the underlying library requirements are here: https://github.com/harthur-org/brain.js/issues/39

**Comment by samparsky** [comment]
@robertleeplummerjr were you able to achieve this?

**Comment by robertleeplummerjr** [comment]
The entire of brain.js v2, which we hope to have a release candidate out by the end of the month, will be the catalyst for this, and I have been working on it for essentially a couple years.  I wanted to not just do it, but make it so easy a child could do it.

I was told:

>Fun, but very hard!

I want to tell my future self, future @wagenaartje, and future @samparsky: "ha, easy"

We will see, but I know [the api](https://github.com/BrainJS/brain.js/wiki/Roadmap), is working, and will give us the wings to achieve this.

**Comment by idavillc** [comment]
I was wondering if the where's waldo generator was completed?

**Comment by Hawzen** [comment]
That would be very cool
